ipn/localapi/localapi.go

```diff
@@ -502,7 +502,10 @@ func (h *Handler) servePprof(w http.ResponseWriter, r *http.Request) {
 }
 
 func (h *Handler) serveServeConfig(w http.ResponseWriter, r *http.Request) {
-	if !h.PermitWrite {
+	// Don't require PermitWrite for GET, since it's read-only.
+	// Commands like "tailscale status" read the config,
+	// and we don't want to require elevated perms for that.
+	if (r.Method == "GET" && !h.PermitRead) || !h.PermitWrite {
```

Comment:
This isn't right. For a GET it'll match the second part (not permit write) and fail.

Move the checks down in the switch below and add a default case for invalid method

Reply:
Doh. Done and cleaned up the json header being sent always.